java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -19,9 +19,11 @@
 /** Module for JSON commands. */
 public class Json {
 
-    public static final String JSON_PREFIX = "JSON.";
+    private static final String JSON_PREFIX = "JSON.";
     public static final String JSON_SET = JSON_PREFIX + "SET";
     public static final String JSON_GET = JSON_PREFIX + "GET";
```

Comment:
Change both `GET` and `SET` to private too?

Reply:
It is used somewhere in UT.